docs/serverless-v2/08-02-expose-lambda.md

```diff
@@ -69,3 +75,33 @@ Follows these steps:
     ```bash
     curl https://$NAME.$DOMAIN
     ```
+
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+> **NOTE:** The API Rules functionality that allow you to expose lambdas is an experimental feature, and it is not enabled by default in the Console UI. To use it, enable **Experimental functionalities** in the **General Settings** view before you follow the steps.
+
+1. Select a Namespace from the drop-down list in the top navigation panel where there is an existing lambda which you want to expose through an API Rule.
+2. Go to the **API Rules [preview]** view at the bottom of the left navigation panel and select **Add API Rule**.
+3. In the **General settings** section:
+
+    - Enter API Rule's **Name** matching the lambda's name.
+
+    >**NOTE:** The APIRule CR can have a different name than lambda, but it is recommended that all related resources share a common name.
+
+    - Enter **Hostname** on which you want to expose lambda.
+
+    - Select the lambda from the drop-down list in the **Service** column.
```

Comment:
or `under **Service**`? as in `under **Host**` a bit below.

Reply:
I think I would leave "in the xxx column".

Reply:
👍